randovania/games/prime3/logic_database/Bryyo - Fire.txt

```diff
@@ -140,6 +140,8 @@ Extra - asset_id: 4215957264467390882
               Any of the following:
                   After Imperial Hall Grapple Lasso Walls or Bomb/Spring Space Jump (Advanced)
                   Boost Ball and Bomb/Spring Space Jump (Intermediate)
+                  # https://www.youtube.com/watch?v=Z-XICpMJWoE
+                  Before Imperial Hall Grapple Lasso Walls and Movement (Beginner) and Standable Terrain (Beginner)
```

Comment:
I would need someone more experienced than me to remind me, but I'm not sure we need the event check here.
From what I've been told, having  `Before (event)` checks are pretty bad for generator health and should be avoided as much as possible.
Think in this scenario you should be fine to just remove it since SJ is already required in this connection

Reply:
You should only but in `before whatever` if  having done or collected `whatever` makes your condition harder or impossible.  If it's not essential to have this option here, it might be worth it to forgo having it, just to avoid making `whatever` dangerous. Also worth mentioning, in Dread, we have an option in the generation section for highly dangerous logic, to cordon off this kind of dangerous logic. The point being that dangerous resources as bad for both the generator, resolver and the player experience.

For this case I was a bit confused by how video beings at the floor of the room when the connection is from Front of Lasso Walls. I think the before clause is redundant in this case, since this stuff would fit into `after walls event or here` branch. Though, the lasso event is already dangerous because of the connection from Door to Fiery Airdock to Front of Lasso Walls.

Reply:
the other question that I have: can you still do this standable after you have shoot the thingy so that it moves out of the way?

Reply:
Yes, it still works, just tested it